### PR TITLE
[PREVIEW] PRO-3083: Reorder TaskList items

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by shepda** [comment]
Please be careful with your commit messages...`PRO-3083: WIP` doesn't provide any description of what changes the commit makes

**Comment by davejones74** [comment]
HI @LucaDelBuonoHMCTS a fix to e2e tests has broken accessibility. This was working on the 6th September at 11:30 but but the following commit at 16:55 broke the accessibility tests.
https://build.platform.hmcts.net/job/HMCTS/job/probate-frontend/view/change-requests/job/PR-279/

Commit causing issues.
https://build.platform.hmcts.net/job/HMCTS/job/probate-frontend/view/change-requests/job/PR-279/12/changes

File: app/journeys/probate.js [hunk 1]
```diff
@@ -160,9 +163,17 @@ const stepList = {
     CopiesOverseas: 'CopiesSummary',
     CopiesSummary: 'TaskList',
 
+    // Payment Task -----------------------------------------------------------
```
Comment (davejones74) [inline app/journeys/probate.js]: We don't use comments in the code as the code should be easy to read. Can you remove these comments.

**Comment by LucaDelBuonoHMCTS** [comment]
Splitting this PR according to 5 separate tickets

**Pull request closed without merging**